### ISSUE TITLE
Switch to SAMCategories for color parsing

### ISSUE THREAD

**Issue by mertdumenci** [clayallsopp/HybridKit-iOS#4]
@soffes wrote an awesome color parser, definitely better than ours which we hacked together. I'll make the switch tomorrow!

https://github.com/soffes/SAMCategories/blob/master/SAMCategories/UIKit/UIColor%2BSAMAdditions.h

**Comment by clayallsopp** [comment]
:+1: looks like we should `sam_colorWithCSS`, right?

**Comment by mertdumenci** [comment]
Yeah. Much better than our runtime hack for `-[UIColor XColor` stuff.

**Comment by soffes** [comment]
:heart:
